src/core/EdFi.DataManagementService.Core.Tests.Unit/Middleware/ExtractDocumentInfoMiddlewareTests.cs

```diff
@@ -62,13 +62,7 @@ public async Task Setup()
 
             context = new(
                 new(
-                    Body: JsonNode.Parse(
-                        """
-                        {
-                            "schoolId": "123"
-                        }
-"""
-                    )!,
+                    Body: """{"schoolId": "123"}"""!,
```

Comment:
You need to put the parsed version of the string body in `context.ParsedBody` just like would happen after your new middleware runs.

Reply:
> You need to put the parsed version of the string body in `context.ParsedBody` just like would happen after your new middleware runs.

Thanks @bradbanister , it was my first solution to the problem, however I thought that this change should be made in many places and that's why I added it in ValidateEqualityConstraintMiddleware and ´ExtractDocumentInfoMiddleware´